src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -118,7 +125,10 @@ def __call__(
                 noise_pred = noise_pred_uncond + guidance_scale * (noise_pred_text - noise_pred_uncond)
 
             # compute the previous noisy sample x_t -> x_t-1
-            latents = self.scheduler.step(noise_pred, t, latents, **extra_step_kwargs)["prev_sample"]
+            if isinstance(self.scheduler, LmsDiscreteScheduler):
+                latents = self.scheduler.step(noise_pred, i, latents, **extra_step_kwargs)["prev_sample"]
+            else:
+                latents = self.scheduler.step(noise_pred, t, latents, **extra_step_kwargs)["prev_sample"]
```

Comment:
This can be made cleaner by removing the branch with `t`->`i`, but for now I'm not touching the timesteps from k-lms for ease of debugging.